android/src/main/java/com/facebook/spectrum/Configuration.java

```diff
@@ -257,6 +271,17 @@ public Builder setUseInterlacing(final Boolean useInterlacing) {
       return this;
     }
 
+    /**
+     * Png: The compression level that is used by zlib to determine
+     * how much time to spend on trying to compress the image data.
+     * 0 is for not using compression at all and 9 is for the best compression.
+     */
+    public Builder setCompressionLevel(final Integer compressionLevel) {
+      Preconditions.checkArgument(compressionLevel >= -1 && compressionLevel <= 9);
```

Comment:
I think we should exclude `-1` here as it's an implementation detail. Otherwise we have both `null` and `-1` resulting in the same behaviour. Also matches the documentation then.